config.go

```diff
@@ -156,3 +178,23 @@ func envProvider() *env.Env {
 			strings.TrimPrefix(s, defaultPrefix)), "_", defaultDelimiter, -1)
 	})
 }
+
+// pemToPrivateKey converts a PEM encoded private key to a Signer interface. It supports EC, RSA and PKIX PEM encoded strings
+func pemToPrivateKey(bytes []byte) (signer crypto.Signer, err error) {
+	key, _ := ssh.ParseRawPrivateKey(bytes)
```

Comment:
why from the `ssh` package? Isn't it a normal EC/RSA PKCS-something encoded key?

Reply:
ssh covers all "normal" encoded keys and the SSH format. So it can handle more